utils/calibration.py

```diff
@@ -100,7 +100,10 @@ def get_kde(tpred_samples, data, target, i, sigmas_samples, position=0, idTest=0
 	else:
 		# TODO: Here also, the coordinates may be absolute or relative
 		# depending on the prediction method
-		sample_kde = tpred_samples[:, i, position, :] + np.array([data[i,:,:][-1].numpy()])
+		if pdf_flag:
```

Comment:
Why is this name "pdf_flag"? Couled be something like absolute/relative?

Reply:
`pdf_flag` is a flag to identify how to compute:
```
if pdf_flag:
	sample_kde = tpred_samples[:, i, position, :]
else:
	sample_kde = tpred_samples[:, i, position, :] + np.array([data[i,:,:][-1].numpy()])
``` 

The refactored `calibration_pdfXX` function computes `sample_kde` as `pdf_flag=True`

Reply:
May be this flag name should reflect better its use; what about absolute_coords_flag ?